### PR TITLE
translate setup/kubernetes/ansible-install/index.md into Chinese

### PR DESCRIPTION
**Pull request by xiaosuiba**
[original code](https://github.com/istio/istio.github.io/tree/master/content/docs/setup/kubernetes/ansible-install/index.md) 
[original page](https://preliminary.istio.io/docs/setup/kubernetes/ansible-install)
@rootsongjc  please review.

### CODE REVIEW

File: content_zh/docs/setup/kubernetes/ansible-install/index.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: 使用 Ansible 安装
+description: 使用内置的 Ansible playbook 安装 Istio
+weight: 40
+keywords: [kubernetes,ansible]
+---
+
```
Comment (gyliu513) [inline content_zh/docs/setup/kubernetes/ansible-install/index.md]: Only need one blank line.

Ditto for others in this document.

**Comment by rootsongjc** [comment]
@xiaosuiba CI failed to fix this please rebase the master.
```
- ./public/docs/tasks/telemetry/fluentd/index.html
  *  213:53: ERROR: Unexpected end tag : p (line 213)
htmlproofer 3.9.1 | Error:  HTML-Proofer found 1 failure!
```

**Pull request merged**